### PR TITLE
WS-010: Login page

### PR DESCRIPTION
**Pull request by chiara-d697**
Connect the login page to the backend, so that users can log in if their information is in the database.
image

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+
+
 
 const Routes: React.FunctionComponent = () => {
   return (
     <Switch>
-      <Route path="/">
-        <Home />
-      </Route>
+      <Route path="/home" component={Home}/>
```
Comment (TimLeach635) [inline frontend/src/App.tsx:15]: Why have you changed the path? I think `/` should always take you to the homepage, no?

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+        
+      <Route exact path="/login" component={Login}/>
     </Switch>
   );
 };
 
 const App: React.FunctionComponent = () => {
+
+
```
Comment (TimLeach635) [inline frontend/src/App.tsx:24]: Have you run `npm install`? I would expect the formatter to have removed these lines, but it seems like it hasn't run for you

File: frontend/src/components/login/Login.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.login, form {
```
Comment (TimLeach635) [inline frontend/src/components/login/Login.scss:1]: You've decided to format _all forms_ in the whole site this way! Was that on purpose?

File: frontend/src/components/login/Login.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+input {
```
Comment (TimLeach635) [inline frontend/src/components/login/Login.scss:8]: Likewise here - have you meant to assert that _every input element on the whole site_ should have a `2em` margin?

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Nice! Just some comments for you

**Comment by doctone** [comment]
In our App.tsx file, we had to change the order of the routes in order to see the login page when we click the nav link:
image

We're not quite sure why this is working, and whether it means our solution is correct or just 'hacked' ?!?!

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**